### PR TITLE
Fix phone field to not change to unsupported countries using the dial code

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-dial-code-change-with-supported-countries](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-dial-code-change-with-supported-countries)

File: src/app/public/modules/phone-field/phone-field.component.ts [hunk 1]
```diff
@@ -365,8 +365,16 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
       let foundCountry = this.countries
         .find(country => country.dialCode === dialCode && country.priority === 0);
 
-      if (foundCountry && foundCountry !== this.selectedCountry) {
-        newCountry = foundCountry;
+      if (foundCountry) {
+        // Ensure that the country that was found in one of our supported countries
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/phone-field/phone-field.component.ts]: Did you mean to say, "...that was found _is_ one of our supported countries"?

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/phone-field/phone-field.component.ts]: Ya. I slightly tweaked the end of that sentence.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-dial-code-change-with-supported-countries](https://developer.blackbaud.com/skyux-phone-field?_sp.spa=pr-dial-code-change-with-supported-countries)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**